### PR TITLE
Use 3.6 instead of 3.5 in float fract() documentation

### PR DESCRIPTION
**Pull request by SOF3**
It is not self-explanatory whether the fract() function inverts the fractional part of negative numbers. This change clarifies this possible question (so that it is `.6` not `.4`)

### CODE REVIEW

**Comment by Dylan-DPC** [comment]
Failed in rollup https://github.com/rust-lang/rust/pull/68242

@bors r-

File: src/libstd/f64.rs [hunk 1]
```diff
@@ -109,10 +109,10 @@ impl f64 {
     /// # Examples
     ///
     /// ```
-    /// let x = 3.5_f64;
-    /// let y = -3.5_f64;
+    /// let x = 3.6_f64;
+    /// let y = -3.6_f64;
     /// let abs_difference_x = (x.fract() - 0.5).abs();
```
Comment (mati865) [inline src/libstd/f64.rs]:
```suggestion
    /// let abs_difference_x = (x.fract() - 0.6).abs();
```

**Comment by mati865** [comment]
GitHub has broke spacing in the last commit, you should make is as a fixup to the previous one.

**Pull request merged**